Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -139,37 +139,52 @@
 
 %% 0.a Manage masking
 if ~isfield(x.S,'bMasking') || isempty(x.S.bMasking)
-    x.S.bMasking = [1 1 1 1]; % default
+	if x.S.bSubjectSpecificROI
+		% For Specific ROIs - Lesion and ROI provided on the input, we do not superimpose additional masks by default
+		fprintf('\n\n%s\n\n', 'Calculating statistics for subject-specific lesions or ROIs -> disabling all masks: susceptibility regions will not be masked, vascular artifacts may be included.');

```

Comment:
I would still make it a warning, otherwise users won't read this. The point is that by default the user can trust the masks, but now there is different behavior between the user-provided ROIs/lesions and the atlas-provided ROIs. Otherwise fine.

Reply:
Fine with me. I just copy-pasted what you wrote :) Done.